OD_data_cleaning/test.output.tsv

```diff
@@ -0,0 +1,6 @@
+name	phase	slope	intercept	N0	SNR	rank

```

Comment:
We should move these data files to a different directory for using test or example data.  You may want to use the croissance directory structure as inspiration for organising the sources code, tests, and examples.